### PR TITLE
binary search

### PR DESCRIPTION
**Pull request by rahuTi**
binary search extended

### CODE REVIEW

File: c1.txt [hunk 1]
```diff
@@ -2,4 +2,43 @@ this is my code...
...
+        int x = 10;
+        int result = ob.binarySearch(arr, x);
+        if (result == -1)
+            System.out.println(
+                "Element is not present in array");
+        else
+            System.out.println("Element is present at "
+                               + "index " + result);
+    }
+}
```
Comment (rahuTi) [inline c1.txt:44]: search

Comment (rahuTi) [inline c1.txt:44]: search

Comment (rahuTi) [inline c1.txt:44]: ssssss

Comment (rahuTi) [inline c1.txt:44]: sssss

Comment (rahuTi) [inline c1.txt:44]: zzzzz

Comment (rahuTi) [inline c1.txt:44]: in which language is this code. also explane what is this code doing with bullet points.

Comment (Rahu09) [inline c1.txt:44]: /explain

Comment (rahuTi) [inline c1.txt:44]: /explain

Comment (rahuTi) [inline c1.txt:44]: /explain

**Pull request closed without merging**